app/src/components/explorer/SearchResultsTable.vue

```diff
@@ -266,6 +253,9 @@ export default {
     })
   },
   methods: {
+    async loadMaterials (selectedValue) {
```

Comment:
If we're changing the table header to properties, we may also want to change these to loadProperties ?

Reply:
I agree